python/moosesqa/check_requirements.py

```diff
@@ -79,7 +79,7 @@ def check_requirements(requirements, file_list=None, color_text=True, **kwargs):
         raise ValueError(msg)
     elif file_list is None:
         root = mooseutils.git_root_dir()
-        file_list = mooseutils.git_ls_files(root, recurse_submodules=False)
+        file_list = mooseutils.git_ls_files(root, recurse_submodules=True)
```

Comment:
@cbolisetti This should fix your MASTODON files not being found. I gave you the wrong file to change this in earlier in the day.